cmd/syncthing/verbose.go

```diff
@@ -102,8 +102,12 @@ func (s *verboseSvc) formatEvent(ev events.Event) string {
 		return fmt.Sprintf("Started syncing %q / %q (%v %v)", data["folder"], data["item"], data["action"], data["type"])
 	case events.ItemFinished:
 		data := ev.Data.(map[string]interface{})
-		if err := data["err"]; err != nil {
```

Comment:
Yeah, we always populate it as "error" in the ItemFinished calls. It was just not tested properly.